independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json

```diff
@@ -0,0 +1,843 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CQC Data",
+    "description": "To retrieve regulator review reports and ratings for England",
+    "version": "10",
+    "contact": {
+      "name": "Martyn Lesbirel",
+      "email": "ipc-author-cqc@outlook.com"
+    }
+  },
+  "host": "apicqcorguk",
```

Comment:
@martynlesbirel It doesn't look like a host URL.

Reply:
Mea Culpa. Now corrected.